### PR TITLE
vumi-go application skeleton and testing creation

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn this one is fairly basic, just skeleton code modified with new ghr refs. An example of when I'm not sure about just merging my own stuff :grinning:

File: js_sandbox/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "keywords": [
+    "vumi",
+    "ussd",
+    "sms"
+  ],
+  "author": "Mike Jones <mike@westerncapelabs.com>",
+  "license": "BSD",
+  "readmeFilename": "README"
+}
```
Comment (smn) [inline js_sandbox/package.json]: new line

File: js_sandbox/test/test-go-nike-ghr.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+
+    // This is an example of a test that we don't want to run at the moment
+    // so we add .skip
+    it.skip('should go to end when asked for them to say someting', function() {
+        var p = tester.check_state({current_state: 'state_we_have_not_made'}, 'Hello world!',
+            'end_state', '^Thank you and bye bye!');
+    });
+
+});
```
Comment (smn) [inline js_sandbox/test/test-go-nike-ghr.js]: new line?

File: js_sandbox/test/test.js [hunk 3]
```diff
@@ -0,0 +1 @@
+require("./test-go-nike-ghr.js")
```
Comment (smn) [inline js_sandbox/test/test.js]: new line!

**Comment by smn** [comment]
:+1: :smile:

**Pull request merged**